### DISCUSSION TITLE
hook syscalls?

### DISCUSSION THREAD

**Discussion by atimofeev86** [LSPosed/LSPosed#2746] [Q&A] [answered]
I really can hook any native code with lsposed, it's perfect (https://github.com/LSPosed/LSPosed/wiki/Native-Hook).
How can I hook syscall?
![image](https://github.com/LSPosed/LSPosed/assets/41478386/a2ace10f-af4c-458e-93a9-49fed4ec2455)

**Comment by yujincheng08** [comment, marked as answer]
seccomp
